modules/amp-yospace/pages/amp-yospace-plugin.adoc

```diff
@@ -0,0 +1,63 @@
+:current-module: amp-yospace
+include::ROOT:partial$amp-gobal-attributes.adoc[]
+:imagesdir: image-catalog
+:amp-plugin-docs: https://developer.yospace.com/sdk-documentation/apple/userguide/latest/en/index-en.html
+:amp-plugin-sample-ios: https://mdtp-a.akamaihd.net/amp-ios-sdk/samples/AmpYoSpaceSample.zip
+:amp-plugin-sample-tvos: https://mdtp-a.akamaihd.net/amp-ios-sdk/samples/AmpYoSpaceSampleTV.zip
+:plugin-common-name:YoSpace
+
+= YoSpace Analytics
```

Comment:
I am confused @mchacong @orlando-amp Miguel, I thought our yospace integration is just for SSAI Ads. 

I am not aware of Yospace Analytics, 
is this file correct?